src/pages/HomePage.jsx

```diff
@@ -1,10 +1,291 @@
-    // eslint-disable-next-line react/no-unescaped-entities
-    <div>Homepage</div>
+    <div>
+      <Container maxW="90vw">
+        <Heading size="lg" py="10">
+          Welcome Back, Peter!
+        </Heading>
+        <Heading size="md" py="1">
+          Notifications
+        </Heading>
+        <VStack spacing="5px" align="left">
+          <Toast
+            title="You have submitted a log for: OC09c, OC09b"
+            description="You have not submitted a log for: OC01"
+            status="info"
+            variant="left-accent"
+            closeButton={false}
+          />
+          <Toast
+            title="Your monitor log for OC09a on 02-16-2022 has been approved!"
+            description="Thank you for your hard work, keep it up!"
+            status="success"
+            variant="left-accent"
+            closeButton
+          />
+          <Toast
+            title="Monitor logs for segment(s) OC01 have not been submitted yet."
+            description="Please submit these logs by 02-28-2022 at 12:00PM."
+            status="warning"
+            variant="left-accent"
+            closeButton
+          />
+          <Toast
+            title="Edits have been requested for your OC09b log on 02-16-2022"
+            description="Request Reason: [Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+           eiusmod tempor incididunt ut labore et dolore magna aliqua. Diam volutpat commodo
+           sed egestas egestas fringilla.]"
+            status="error"
+            variant="left-accent"
+            goToLogButton
+          />
+        </VStack>
+        <Box w="1200px" h="60px" />
+        <Heading size="md">Segment Assignment(s)</Heading>
+        <Text py="3" fontSize="16px" color="#4A5568">
+          Note on Parking: If you pay for parking (not to exceed $6 without approval), please submit
+          your receipts for reimbursement with your name and segment.
+        </Text>
+        <HStack spacing="50px">
+          {userData != null &&
+            userData.segments.map(segment => (
+              <SegAssignment
+                key={segment.segmentID}
+                title={`${segment.name} - ${segment.segmentId}`}
+                place={segment.streets}
+                description={segment.parking}
+              />
+            ))}
+          <SegAssignment
+            title="OC09 &nbsp;&nbsp;&nbsp; Huntington State Beach State Beach 1"
+            place="Beach Blvd to Magnolia"
+            description="Park in Huntington State Beach if you have a pass, or park on Seashore Drive at Orange
+         Street in Newport Beach."
+          />
+          <SegAssignment
+            title="OC01 &nbsp;&nbsp;&nbsp; Huntington State Beach State Beach 1"
+            place="Magnolia to Santa Ana River/Least Tern Preserve"
+            description="Park in Huntington State Beach if you have a pass, or park on Seashore Drive at Orange
+         Street in Newport Beach."
+          />
```

Comment:
Remove these hard coded things since you already finished mapping assignments from `userData`